templates/ort-scan.yml

```diff
@@ -109,8 +109,8 @@
 
       if [ ! -n "${VCS_TYPE+x}" ] && \
         [ ! -n "${VCS_URL+x}" ] && \
-        [ ! -n "${VCS_REVISION}" ] && \
-        [ ! -n "${VCS_PATH}" ]
+        [ ! -n "${VCS_REVISION+x}" ] && \
```

Comment:
What do the `+x` do? The commit message does not explain it

Reply:
I've taken this from here: https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash.
Not sure if it's needed though. What do you think? Shall we just drop it? Note: The two lines above already have the "+x" from one of my preceding changes.